basic_strategies.csv

```diff
@@ -0,0 +1,3 @@
+Defector,Bully,Suspicious Tit For Tat,Alternator,Tit For Tat,Anti Tit For Tat,Win-Stay Lose-Shift,Cooperator
```

Comment:
I don't think you wanted to commit these. They're created by the test suite (a couple of other files are similarly created)?